### ISSUE TITLE
Windows port 

### ISSUE THREAD

**Issue by philippk80** [roc-project/roc#132]
I think the title says it all. This issue is for tracking/discussing efforts and  informations  about this topic.
Like @gavv mentioned in #131 this isn't planed and labeled to a milestone yet.

**Comment by philippk80** [comment]
@gavv one question about scons
I searched hours for documentation in scons about this but without effort.
Is env.Which() similar to https://docs.python.org/3/library/shutil.html?highlight=.which#shutil.which ?
Does it search in the Path directory for the given keyword? For example env.Which("python") will search for the python.exe?

**Comment by philippk80** [comment]
Okay figured it out! This took me long.
Is this predefined in scons that the site_scons stuff will be used?
Didn't found any reference in the SConstruct.
This was hard without knowing scons. 😆

**Comment by gavv** [comment]
You're right, this function is defined in our SCons plugin (site_scons/site_tools/roc).

* here is the function itself: https://github.com/roc-project/roc/blob/master/site_scons/site_tools/roc/helpers.py#L77

* here we register it in the environment: https://github.com/roc-project/roc/blob/master/site_scons/site_tools/roc/helpers.py#L435

* here is the place in SConstruct where we enable our plugin: https://github.com/roc-project/roc/blob/master/SConstruct#L36

> Does it search in the directorys given in the PATH variable in kombination with the PATHEXT for the given keyword? For example env.Which("python") will search for the python.exe?

Yes, it's supposed to do so.

**Comment by gavv** [comment]
Related: #143, #144.

**Comment by gavv** [comment]
I've created #292 sub-task.
